x/concentrated-liquidity/lp_test.go

```diff
@@ -808,21 +839,39 @@ func (s *KeeperTestSuite) TestUpdatePosition() {
 
 	tests := map[string]updatePositionTest{
 		"update existing position with positive amount": {
-			poolId:                    1,
-			ownerIndex:                0,
-			lowerTick:                 DefaultLowerTick,
-			upperTick:                 DefaultUpperTick,
-			joinTime:                  DefaultJoinTime,
-			positionId:                DefaultPositionId,
-			liquidityDelta:            DefaultLiquidityAmt,
-			amount0Expected:           DefaultAmt0Expected,
+			poolId:         1,
+			ownerIndex:     0,
+			lowerTick:      DefaultLowerTick,
+			upperTick:      DefaultUpperTick,
+			joinTime:       DefaultJoinTime,
+			positionId:     DefaultPositionId,
+			liquidityDelta: DefaultLiquidityAmt,
+			// Note: rounds up in favor of the pool.
+			amount0Expected:           DefaultAmt0Expected.Add(sdk.OneInt()),
 			amount1Expected:           DefaultAmt1Expected,
 			expectedPositionLiquidity: DefaultLiquidityAmt.Add(DefaultLiquidityAmt),
 			expectedTickLiquidity:     DefaultLiquidityAmt.Add(DefaultLiquidityAmt),
 			expectedPoolLiquidity:     DefaultLiquidityAmt.Add(DefaultLiquidityAmt),
 			numPositions:              1,
 			expectedError:             false,
 		},
+		"update existing position with negative amount": {
+			poolId:         1,
+			ownerIndex:     0,
+			lowerTick:      DefaultLowerTick,
+			upperTick:      DefaultUpperTick,
+			joinTime:       DefaultJoinTime,
+			positionId:     DefaultPositionId,
+			liquidityDelta: DefaultLiquidityAmt.Neg(), // negative
+			// Note: rounds down in favor of the pool.
+			amount0Expected:           DefaultAmt0Expected.Neg(),
+			amount1Expected:           DefaultAmt1Expected.Neg(),
```

Comment:
Shouldn't we need to round down somewhere?

Reply:
It does round down instead of up with the case of positive liquidity